app/person.rb

```diff
@@ -1,16 +1,17 @@
 require_relative 'nameable'
 class Person < Nameable
+  attr_accessor :rentals, :name, :age
+  attr_reader :id
+
   def initialize(age, name = 'Unknown', parent_permission: true)
     super()
+    @rentals = []
     @id = rand(100_000)
     @age = age
     @name = name
     @parent_permission = parent_permission
   end
```

Comment:
Kindly include a method for adding rentals. Check out an example [here](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship).